### DISCUSSION TITLE
Filter modal stuck

### DISCUSSION THREAD

**Discussion by LemonPie22** [filamentphp/filament#8326] [Help] [answered]
This is an odd one.

I have a resource for quotes that has several filters. When I click on the filter button the modal pops up but it is often stuck, meaning I can't scroll up or down to access other filter options.

<img width="808" alt="Screenshot 2023-09-07 at 11 37 06" src="https://github.com/filamentphp/filament/assets/107488019/70ca8349-9e71-4254-92c2-a388dfbf0a57">

I'm using a MacBook with touch pad and mouse. I've tried all sorts of key combinations (tab, etc.). I've also tried switching browsers (Chrome and Safari) but, right now, I can't get it to budge.

Any clues or workarounds please? The only way I've managed to resolve it is by changing the size of my browser window.

I'm on Filament 2 and my filter  code is as follows:

```
->filters(
    [
        Tables\Filters\TrashedFilter::make(),
        Tables\Filters\SelectFilter::make('organisation_id')
            ->relationship('Organisation', 'name')
            ->label('Customer'),
        Tables\Filters\SelectFilter::make('author_id')
            ->relationship('author', 'name')
            ->label('Prepared by'),
        Tables\Filters\SelectFilter::make('quote_type_id')
            ->relationship('quoteType', 'name')
            ->label('Type'),
        Tables\Filters\SelectFilter::make('status')->options([
            'draft' => 'Draft',
            'review' => 'Review',
            'published' => 'Published',
            'accepted' => 'Accepted',
            'rejected' => 'Rejected',
        ]),
    ]
)
```

**Comment by leandrocfe** [comment, marked as answer]
Not sure why. Maybe you can change some options as

https://filamentphp.com/docs/2.x/tables/filters/#appearance

https://filamentphp.com/docs/2.x/tables/filters/#displaying-filters-above-or-below-the-table-content

**Reply by LemonPie22** [reply]
Thanks @leandrocfe. I decided to render the filters above the table content instead of in a popover.
